hackIDE_project/settings.py

```diff
@@ -23,7 +23,8 @@
 SECRET_KEY = '+h*i@$52+w(_e#etvzgnkjq!q0ajz1qpgs-y9%89x4w3nlct=m'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = (os.environ.get('HACKIDE_DEBUG') != None)
+DEBUG = True
```

Comment:
What's the issue with this? We need to fix it if it doesn't work with `DEBUG = False`. In production, it should always have `DEBUG = False`